### PR TITLE
Feat(System backup) Support longhorn system backup/restore

### PR DESCRIPTION
**Pull request by smallteeths**
https://github.com/longhorn/longhorn/issues/4855
截屏2022-11-22 下午6 09 38

Signed-off-by: Siye Wang <siye@rancher.com>

### CODE REVIEW

**Review by innobead — CHANGES_REQUESTED** [review]
In genreal, LGTM.

File: src/routes/systemBackups/systemBackupsBulkActions.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+function bulkActions({ selectedRows, deleteSystemBuckups, createSystemBuckup }) {
+  const handleClick = (action) => {
+    switch (action) {
+      case 'create':
+        createSystemBuckup()
+        break
+      case 'delete':
+        confirm({
+          title: `Are you sure you want to delete System Buckups ${selectedRows.map(item => item.name).join(',')} ?`,
```
Comment (c3y1huang) [inline src/routes/systemBackups/systemBackupsBulkActions.js:16]:
```suggestion
          title: `Are you sure you want to delete System Backups ${selectedRows.map(item => item.name).join(',')} ?`,
```

File: src/routes/systemBackups/systemBackupsBulkActions.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          </div>
+        )
+      }) }
+    </div>
+  )
+}
+
+bulkActions.propTypes = {
+  selectedRows: PropTypes.array,
+  deleteSystemBuckups: PropTypes.func,
```
Comment (c3y1huang) [inline src/routes/systemBackups/systemBackupsBulkActions.js:46]: deleteSystemBuckups

File: src/routes/systemBackups/systemBackupsBulkActions.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  createSystemBuckup: PropTypes.func,
```
Comment (c3y1huang) [inline src/routes/systemBackups/systemBackupsBulkActions.js:47]: createSystemBackup

**Review by c3y1huang — CHANGES_REQUESTED** [review]

File: src/routes/systemBackups/createSystemRestore.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            ],
+          })(<Input disabled={true} />)}
+        </FormItem>
+        <FormItem label="System Backup" hasFeedback {...formItemLayout}>
+          {getFieldDecorator('systemBackup', {
+            initialValue: item.systemBackup,
+            rules: [
+              {
+                required: true,
+                message: 'Please input System backup image.',
```
[the remark this answers is not in the document]
Comment (smallteeths) [reply src/routes/systemBackups/createSystemRestore.js]: done

File: src/utils/page.js [hunk 5]
```diff
@@ -1,4 +1,18 @@
 export function pagination(key) {
+  if (key === 'systemBackupSize' || key === 'systemBackupSize') {
```
[the remark this answers is not in the document]
Comment (smallteeths) [reply src/utils/page.js]: done

File: src/routes/systemBackups/systemBackupsBulkActions.js (hunk 1 above)
Comment (smallteeths) [reply src/routes/systemBackups/systemBackupsBulkActions.js]: done

File: src/routes/systemBackups/systemBackupsBulkActions.js (hunk 2 above)
Comment (smallteeths) [reply src/routes/systemBackups/systemBackupsBulkActions.js]: done

File: src/routes/systemBackups/systemBackupsBulkActions.js (hunk 3 above)
Comment (smallteeths) [reply src/routes/systemBackups/systemBackupsBulkActions.js]: done

**Review by innobead — APPROVED** [review]

**Pull request merged**